lib/config/fileconf.go

```diff
@@ -96,6 +96,9 @@ type FileConfig struct {
 
 	// Plugins is the section of the config for configuring the plugin service.
 	Plugins PluginService `yaml:"plugin_service,omitempty"`
+
+	// AccessGraph is the section of the config describing AccessGraph service
+	AccessGraph AccessGraph `yaml:"access_graph,omitempty"`
```

Comment:
Nit: I don't think "omitempty" does anything here tbh but I see other fields have it also so 🤷

Reply:
Even if it's pointless at least it's consistent :sweat_smile: